maintenance/wikia/migrateImagesBetweenSwiftDC.php

```diff
@@ -12,17 +12,17 @@
  * Maintenance script class
  */
 class MigrateImagesBetweenSwiftDC extends Maintenance {
-	
+
 	/* @private Int - number of images to sync */
 	private $mLimit;
 	/* @private String - destination DC */
 	private $mDC_dst;
 	/* @private Array - containers */
-	private $source_container;
-	private $dest_container;
-	/* @private \Queue object  */
+	private $source_container = [];
+	private $dest_container = [];
```

Comment:
As a sidenote: `src` and `dst` prefixes work better and most engineers should be familiar with them as `tcpdump` uses them, too :-)